client-stubs/rem.py

```diff
@@ -0,0 +1,33 @@
+# Import the client stub package.
+import rewiringamerica_rem
+
+# Set up your key to send authorized requests to the API. If you
+# don't already have one, you can sign up for one here: https://rewiring.link/api-signup
+configuration = rewiringamerica_rem.Configuration(
+    access_token="key_example"
+)
+
+# Initialize the API client and query parameters to pass into your API request.
+with rewiringamerica_rem.ApiClient(configuration) as api_client:
+    # Create an instance of the API class.
+    api_instance = rewiringamerica_rem.ResidentialElectrificationModelApi(api_client)
+    # Define your the upgrade you want to model, your home's heating fuel,
+    # and your home's address.
+    upgrade = rewiringamerica_rem.SupportedUpgrade.HVAC__HEAT_PUMP_SEER15_HSPF9
+    heating_fuel = rewiringamerica_rem.HeatingFuel.NATURAL_GAS
+    address = "address_example"
+
+    try:
+        # Hit the API endpoint.
+        api_response = api_instance.get_by_address(upgrade, address, heating_fuel)
+        # Extract the total values from the response object.
+        totals = api_response.fuel_results["total"]
+        # Extract the average annual cost savings from the response object.
+        # This is an absolute value because the change in cost, which
+        # decreases, could result in a negative sign.
+        cost_savings = round(abs(totals.delta.cost.mean.value))
```

Comment:
```suggestion
        cost_savings = round(-totals.delta.cost.mean.value)
```
Using an absolute value here makes cost increases and decreases look the same, which is not what we want users to do!

Negating it also matches what we do in other demos (e.g. [here](https://github.com/rewiringamerica/api_demos/blob/499651afcc3f539625555036599149d156f416d8/react/src/App.js#L69)).

Same comment for the Typescript example.